docker-compose.yml

```diff
@@ -1,32 +1,46 @@
 version: "3.9"
 
+# rotating logs, so they do not overflow
```

Comment:
Please remove this section

Reply:
can I keep them for the new components in docker-compose.minio.yaml?

Reply:
yes